docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -95,6 +95,14 @@ Canvas supports [data links](https://grafana.com/docs/grafana/latest/panels-visu
 1. Hover over the element to display the data link tooltip.
 1. Click on the element to be able to open the data link.
 
-If multiple elements use the same field, you can create a unique field name using the [add field from calculation transform](https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation). The alias you create in the transformation will appear as a field you can use with an element.
+
+If multiple elements use the same field name, and you want to control which elements display the data link, you can create a unique field name using the [add field from calculation transform](https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation). The alias you create in the transformation will appear as a field you can use with an element.
+
+1. In the panel editor for the canvas panel, click the **Transform** tab.
+1. Click **+ Add transformation** and select **Add field from calculation**.
```

Comment:
I don't know if this is a new nav change or not, but when I look set up my first transformation, I don't have to click the button; it just shows me a list of transformations to choose from. It's only after I've added one that the button is displayed. Is that the same for you? If so, then I suggest for this sentence:

```suggestion
1. Select **Add field from calculation** from the list of transformations, or click **+ Add transformation** to display the list first.
```